aztec_backend_wasm/src/lib.rs

```diff
@@ -42,32 +44,46 @@ pub fn compute_witnesses(
     // Now use the partial witness generator to fill in the rest of the witnesses
     // which are possible
 
+    use barretenberg_wasm::Plonk;
     let plonk = Plonk;
+    let num_witnesses = circuit.num_vars();
```

Comment:
Nitpick: I don't think we need the `use` when the only line that uses Plonk is line 48